views/js/review/plugins/navigation/review-panel/panel-data.js

```diff
@@ -43,15 +50,32 @@ define([
      */
     const compareByPosition = (a, b) => a.position - b.position;
 
+    /**
+     * Gets the type for a particular item
+     * @param {mapEntry} item
+     * @returns {String}
+     */
+    const getItemType = item => {
+        if (item.informational) {
+            return 'info';
+        }
+        if (item.maxScore) {
+            if (item.score === item.maxScore) {
+                return 'correct';
+            } else {
+                return 'incorrect';
+            }
+        }
+        return 'default';
```

Comment:
Here type of item without maxScore is 'default' (see another comment)

Reply:
Yes, without max score, we cannot know if it is correct or incorrect, so the status is unknown.
However, this is a edge case, as the max score needs to be set for the test review to properly work

Reply:
Yes, without max score, we cannot know if it is correct or incorrect, so the status is unknown.
However, this is a edge case, as the max score needs to be set for the test review to properly work